modules/layers/src/solid-polygon-layer/solid-polygon-layer.js

```diff
@@ -87,9 +94,29 @@ export default class SolidPolygonLayer extends Layer {
         transition: ATTRIBUTE_TRANSITION,
         accessor: 'getPolygon',
         update: this.calculatePositions,
-        noAlloc
+        shaderAttributes: {
+          positions: {
+            size: 3
```

Comment:
Is the `size` override necessary?

Reply:
It's not, though having a 'positions' key is necessary. I figure this is a bit clearer than having an empty object?